### ISSUE TITLE
error starting main.py

### ISSUE THREAD

**Issue by frankcell** [Tkd-Alex/Telegram-InstaPy-Scheduling#27]
`Traceback (most recent call last):
  File "main.py", line 9, in <module>
    from telegram.ext import Updater, CommandHandler, Job, CallbackQueryHandler
  File "/usr/local/lib/python3.5/dist-packages/telegram/ext/__init__.py", line 21, in <module>
    from .dispatcher import Dispatcher, DispatcherHandlerStop, run_async
  File "/usr/local/lib/python3.5/dist-packages/telegram/ext/dispatcher.py", line 33, in <module>
    from telegram import TelegramError
ImportError: cannot import name 'TelegramError'
`

**Comment by Tkd-Alex** [comment]
Install all requirements in .txt file.
```sh
pip install -r requiremenets.txt
```

**Comment by gHashTag** [comment]
After 
`pip install -r requiremenets.txt`

```Successfully installed GitPython-2.1.11 certifi-2018.8.13 clarifai-2.2.3 emoji-0.5.0 future-0.16.0 gitdb2-2.0.4 grpcio-1.17.0 python-telegram-bot-10.1.0 requests-2.19.1 selenium-3.14.0 smmap2-2.0.4 telegram-0.0.1 urllib3-1.23```

but after `python main.py` error:
```Traceback (most recent call last):
  File "main.py", line 9, in <module>
    from telegram.ext import Updater, CommandHandler, Job, CallbackQueryHandler
  File "/usr/local/lib/python2.7/site-packages/telegram/ext/__init__.py", line 21, in <module>
    from .dispatcher import Dispatcher, DispatcherHandlerStop, run_async
  File "/usr/local/lib/python2.7/site-packages/telegram/ext/dispatcher.py", line 33, in <module>
    from telegram import TelegramError
ImportError: cannot import name TelegramError```

**Comment by Tkd-Alex** [comment]
Are you sure to launch __pip3__ and __python3__ ?
Anyway, this issue il related to: https://github.com/python-telegram-bot/python-telegram-bot/issues/395

**Comment by giditre** [comment]
Hi there, same issue here. I just started over on a new VM and used explicitly pip2 so as to be sure about the version I was calling.

`pip2 install -r requirements.txt
[...]
Successfully installed EasyProcess-0.2.5 GitPython-2.1.11 MeaningCloud-python-1.1.1 PyVirtualDisplay-0.2.1 asn1crypto-0.24.0 certifi-2018.11.29 cffi-1.11.5 chardet-3.0.4 clarifai-2.5.1 configparser-3.5.0 cryptography-2.4.2 emoji-0.5.1 enum34-1.1.6 functools32-3.2.3.post2 future-0.17.1 futures-3.2.0 gitdb2-2.0.4 googleapis-common-protos-1.5.6 grpcio-1.17.1 idna-2.8 ipaddress-1.0.22 jsonschema-2.6.0 plyer-1.4.0 protobuf-3.6.1 pycparser-2.19 python-telegram-bot-10.1.0 regex-2018.11.22 requests-2.21.0 selenium-3.141.0 setuptools-20.7.0 six-1.12.0 smmap2-2.0.4 telegram-0.0.1 urllib3-1.24.1`

but then

`python2 main.py
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    from telegram.ext import Updater, CommandHandler, Job, CallbackQueryHandler
  File "/home/ubuntu/.local/lib/python2.7/site-packages/telegram/ext/__init__.py", line 21, in <module>
    from .dispatcher import Dispatcher, DispatcherHandlerStop, run_async
  File "/home/ubuntu/.local/lib/python2.7/site-packages/telegram/ext/dispatcher.py", line 33, in <module>
    from telegram import TelegramError
ImportError: cannot import name TelegramError`

Also, I tried everything by invoking pip3 instead (and python3 afterwards) but got the same error:

`Traceback (most recent call last):
  File "main.py", line 9, in <module>
    from telegram.ext import Updater, CommandHandler, Job, CallbackQueryHandler
  File "/home/ubuntu/.local/lib/python3.5/site-packages/telegram/ext/__init__.py", line 21, in <module>
    from .dispatcher import Dispatcher, DispatcherHandlerStop, run_async
  File "/home/ubuntu/.local/lib/python3.5/site-packages/telegram/ext/dispatcher.py", line 33, in <module>
    from telegram import TelegramError
ImportError: cannot import name 'TelegramError'`

Issue#395 linked above suggests the problem could be in the telegram library, though that library is in the requirements.txt list, isn't it? Sorry I'm confused.

Thanks for any help, I truly appreciate your efforts.
